react-example/src/views/Embedded.tsx

```diff
@@ -148,13 +209,30 @@ export const EmbeddedMessage: FC<Props> = () => {
         required
       />
       <br />
-      {eventsList.map((element: any) => (
```

Comment:
should also use a type other than `any` here.

Reply:
Resolved